### PR TITLE
Landing Page Video Modal

### PR DESCRIPTION
**Pull request by McCambley**
### Edit: Much of what I gripe about below arose as conflicts and changes when I needed to revert to an old commit of Colin's. This led me to submit a PR that indicated that some of Colin's changes were to be considered in mine. Upon submission of this PR, it looks like it's been recognized that those changes have already been approved and merged into `main`. I'll keep the notes below, but know that github has resolved anything that may have looked confusing and all is well now. 

# Describe your changes 

_THE TRIALS AND TRIBULATIONS I HAVE GONE THROUGH TO RESOLVE THIS GIT CONFLICT. Long story short... I added a too-large video file and let it pass through a few commits before learning it was too large for github. I had to call on the git wizards to help remove said file from many commits, then re-relate now unrelated histories that now do not contain a large video file. Things work, but git is scary. That is to say... y'all... please run this locally let me know if things are amiss. I tested vigorously but I git is a scary beast and I want make sure I didn't miss anything._ 

## **But that is not the point of this Pull Request!**

This PR contains a video modal for the landing page. I've updated some styling on the existing Image of Igor that was present from a previous landing page PR. This modal opens when you click the image of igor, then opens a video player. Click the overlay or the x-button to close the video. 

[Here's a demonstration
](https://www.loom.com/share/2968e95638d048ccb5bd6541fb8407e5)

[Trello Card](https://trello.com/c/pAOKsMlA)

## Future Steps:
1) Close the overlay with `ESC`
2) Toggle play with `SPACE`
3) Realign some absolutely positioned elements and adjust some margins on the landing page ahead of Monday's Demo.
4) Combine the `LandingForm` and `Landing` components into a cohesive composition.
5) Use a video more relevant to the project
6) Mobile design (When it's ready)

# I follow best practices 

- [] My code is covered by unit tests - _N/A_
- [] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests) - _Due to some git issues some old approved PR code got roped in and thus this PR may appear more bloated than it actually is (If those diffs show up when comparing these branches, which they may not).
- [] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) - _I can't confidently say this PR is easy to read due to all the differences in commit histories. That is not cause for concern, things are working._
- [x] I have tested my PR locally and proven it works. 
- [x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

**Comment by rileydanejohnston** [comment]
Lot's of great work here. I see a few small style issues and the video doesn't pause when clicked (on my end at least). Take a look at my comments and see what you think.

**Review by rileydanejohnston — COMMENTED** [review]
Test 2

File: frontend/src/components/landing/styledLanding.js [hunk 1]
```diff
@@ -138,3 +142,82 @@ export const Phone = styled.div`
...
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.5);
+  transition: visibility 0.3s ease, opacity 0.3s ease;
+`;
+
+export const VideoWrapper = styled.div`
+  position: relative;
+  width: 331px;
+  height: 586px;
```
[the remark this answers is not in the document]
Comment (McCambley) [reply frontend/src/components/landing/styledLanding.js:165]: The video wrapper, which hides overlay, is these dimensions, the video, which is just a stock video, overlaps the container slightly but isn't shown. The wrapper displays the proper dimensions, but I'll keep an eye on this when the real video arrives. Good eye here 👍

File: frontend/src/components/landing/Landing.js [hunk 2]
```diff
@@ -15,20 +15,57 @@ import {
...
+    }
+    setIsOpen(!isOpen);
+  }
+
   return (
-    <>
+    <Container>
+      <VideoModal $show={isOpen} onClick={toggleVideoModal}>
+        <VideoWrapper>
+          <Video ref={videoRef} src={landingVideo} onClick={(e) => e.stopPropagation()} />
```
[the remark this answers is not in the document]
Comment (McCambley) [reply frontend/src/components/landing/Landing.js:52]: No it does not, I do think I said that in the video though, my b. I'll fix that in a later pull request, or perhaps I'll keep the controls as `shown` to improve ux

[the remark this answers is not in the document]
Comment (McCambley) [reply frontend/src/components/landing/Landing.js]: Jk just fixed it 🎉

[the remark this answers is not in the document]
Comment (rileydanejohnston) [reply frontend/src/components/landing/Landing.js]: I misunderstood what you said in the video. You say that when you click the OVERLAY, it pauses the video and closes the modal, which is correct. I thought you meant you could pause the video by clicking on it. My comment here can be disregarded.

**Review by rileydanejohnston — APPROVED** [review]
Looks great!

**Pull request merged**